src/test/java/core/basesyntax/service/strategy/handler/SupplyHandlerTest.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.strategy.handler;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+class SupplyHandlerTest {
+    private final OperationHandler operationHandler = new SupplyHandler();
+
+    @Test
+    void process_validTransaction_noException() {
```

Comment:
noException for me means that you check that some action will not produce an exception, but in reality you chack that logic works fine. So goal of this test is to ensure logic is ok, I suggest to rename this to process_validTransaction_ok or checkIfTransactionProcessingCorrect somethig that will explicitly say that goal is to test logic. ( do same in other tests)